src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -459,6 +471,41 @@ void EnMag_DrawImageRGBA32(Gfx** gfxP, s16 centerX, s16 centerY, u8* source, u32
     *gfxP = gfx;
 }
 
+#if PLATFORM_N64
+// TODO n64dd functions
+s32 func_801C79BC_unknown(void);
+
+// TODO does not match, some regalloc diffs. https://decomp.me/scratch/H8OK5
+void func_80AEEA48_unknown(Gfx** gfxP, s16 arg1, s16 arg2, u32 arg3) {
+    if ((B_80121AE2 != 0) && (func_801C79BC_unknown() != 0)) {
+        Gfx* gfx = *gfxP;
+        s32 temp_a3;
+        s32 temp_t0;
+
+        gDPPipeSync(gfx++);
+        gDPSetCycleType(gfx++, G_CYC_1CYCLE);
+        // G_RM_XLU_SURF ?
+        gDPSetRenderMode(gfx++, GBL_c1(G_BL_CLR_IN, G_BL_A_IN, G_BL_CLR_MEM, G_BL_1MA),
+                         GBL_c2(G_BL_CLR_IN, G_BL_A_IN, G_BL_CLR_MEM, G_BL_1MA) | IM_RD | CVG_DST_FULL | ZMODE_OPA |
+                             FORCE_BL);
```

Comment:
This appears to be `gDPSetRenderMode(gfx++, G_RM_XLU_SURF, G_RM_XLU_SURF2)`

Reply:
I'm waiting for the function to match to consider cleanups, though this one is pretty safe I guess